app/views/activities/_activity.html.erb

```diff
@@ -1,17 +1,36 @@
-<% if details %>
-  <a class="activity list-group-item t-activity" href="<%= edit_appointment_path(activity.appointment) %>">
-<% else %>
-  <li class="activity list-group-item t-activity <%= 't-hidden-activity activity--hideable activity--hide' if hide %>" data-activity-id="<%= activity.id %>">
-<% end %>
-    <div class="activity-details">
+<li class="activity list-group-item t-activity
+  <%= 't-hidden-activity activity--hideable activity--hide' if hide %>
+  <% if activity.high_priority? %>
+    t-high-priority-activity
+    t-<%= 'un' if !activity.resolved? %>resolved-activity
+    activity--priority-<%= 'un' if !activity.resolved? %>resolved
+  <% end %>
+" data-activity-id="<%= activity.id %>">
+  <div class="activity__inner">
+    <% if details %>
+      <a href="<%= edit_appointment_path(activity.appointment) %>" class="activity__link">
+    <% end %>
       <%= content_for(:activity_message) %>
-      <%= render('activities/details', appointment: activity.appointment) if details %>
-    </div>
-    <div class="activity-badge">
-      <em class="badge"><%= activity.created_at.in_time_zone('London').to_s(:govuk_date_short) %> (<%= time_ago_in_words(activity.created_at.in_time_zone('London'))%> ago)</em>
-    </div>
-<% if details %>
-  </a>
-<% else %>
-  </li>
-<% end %>
+    <% if details %>
+      </a>
+    <% end %>
+    <%= render('activities/details', appointment: activity.appointment) if details %>
+    <span class="activity__date">
+      <%= activity.created_at.to_s(:govuk_date_short) %> (<%= time_ago_in_words(activity.created_at.in_time_zone('London'))%> ago)
```

Comment:
This needs to be displayed with respect to the London time zone too. Sorry, missed this before.